### ISSUE TITLE
Linear SVM should return true distance to hyperplane

### ISSUE THREAD

**Issue by ejolly** [cosanlab/nltools#265]
Apparently `sklearn` doesn't normalize distance to hyperplane, which is needed to make it interpretable and to compare across datasets/models (not talking about probabilities at all here), because the calculation is entirely dependent on the kernel you use. A linear kernel however makes the transform easy and aids the interpretation, as it's normalized to the range 1, see [here](https://stackoverflow.com/questions/44579826/does-the-decision-function-in-scikit-learn-return-the-true-distance-to-the-hyper) and [here](https://stackoverflow.com/questions/11030253/decision-values-in-libsvm). The reason for this is the underlying libsvm library does not do this normalization for you automatically. This results in differences if you use libsvm in other contexts (e.g. Matlab); results may not agree. 

We can return this value instead in the specific case of a linear kernel with an SVM. 

```
svc = SVC(kernel='linear')
y = svc.decision_function(x)
w_norm = np.linalg.norm(svc.coef_)
dist = y / w_norm
```

**Comment by ejolly** [comment]
Closing as obsolete. The `dist_from_hyperplane` prediction output was removed in the v0.6.0 `predict` rewrite — `predict` now returns cross-validated scores plus a back-projected weight map, so there is no raw `decision_function` output to normalize by `y/‖w‖`. If normalized decision values are still wanted, they'd be a new feature on the redesigned predict API.
